CMake/config/ReleaseDebugAutoFlags.cmake

```diff
@@ -31,19 +31,16 @@ endif()
 # Debug : Optimized for debugging, include symbols
 # Release : Release mode, no debuginfo 
 # RelWithDebInfo : Distribution mode, basic optimizations for potable code with debuginfos
-# Fast : Maximum level of optimization. Target native architecture, not portable code
 
 include(CompilerFlagsHelpers)
 
 
-set(CMAKE_C_FLAGS_RELEASE  "${CMAKE_C_WARNING_ALL} ${CMAKE_C_OPT_NORMAL}")
-set(CMAKE_C_FLAGS_DEBUG  "${CMAKE_C_DEBUGINFO_FLAGS} ${CMAKE_C_WARNING_ALL} ${CMAKE_C_WARNING_DEBUG} ${CMAKE_C_OPT_NONE} ${CMAKE_C_STACK_PROTECTION}")
-set(CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_DEBUGINFO_FLAGS} ${CMAKE_C_WARNING_ALL} ${CMAKE_C_WARNING_DEBUG} ${CMAKE_C_OPT_NORMAL}")
-set(CMAKE_C_FLAGS_FAST "${CMAKE_C_WARNING_ALL} ${CMAKE_C_OPT_FASTEST} ${CMAKE_C_LINK_TIME_OPT} ${CMAKE_C_GEN_NATIVE}")
+set(CMAKE_C_FLAGS_RELEASE  "${CMAKE_C_WARNING_ALL}")
+set(CMAKE_C_FLAGS_DEBUG  "${CMAKE_C_WARNING_ALL} ${CMAKE_C_WARNING_DEBUG}")
+set(CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_WARNING_ALL} ${CMAKE_C_WARNING_DEBUG}")
 
 
 
-set(CMAKE_CXX_FLAGS_RELEASE  "${CMAKE_CXX_WARNING_ALL} ${CMAKE_CXX_OPT_NORMAL}")
-set(CMAKE_CXX_FLAGS_DEBUG  "${CMAKE_CXX_DEBUGINFO_FLAGS} ${CMAKE_CXX_WARNING_ALL} ${CMAKE_CXX_WARNING_DEBUG} ${CMAKE_CXX_OPT_NONE} ${CMAKE_CXX_STACK_PROTECTION}")
-set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_DEBUGINFO_FLAGS} ${CMAKE_CXX_WARNING_ALL} ${CMAKE_C_WARNING_DEBUG} ${CMAKE_CXX_OPT_NORMAL}")
-set(CMAKE_CXX_FLAGS_FAST "${CMAKE_CXX_WARNING_ALL} ${CMAKE_CXX_OPT_FASTEST} ${CMAKE_CXX_LINK_TIME_OPT} ${CMAKE_CXX_GEN_NATIVE}")
+set(CMAKE_CXX_FLAGS_RELEASE  "${CMAKE_CXX_WARNING_ALL}")
```

Comment:
`CMAKE_CXX_FLAGS_RELEASE` already come with some good defaults right? Maybe worth to just "add" our extra flags, but not overwrite the defaults